libraries/bot-builder/src/main/java/com/microsoft/bot/builder/skills/SkillHandlerImpl.java

```diff
@@ -83,6 +97,9 @@ public SkillHandlerImpl(
         return processActivity(claimsIdentity, conversationId, activityId, activity);
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
This class is not extending from another.. so the `inheritDoc` it's not okay. We should populate the commentaries.